charts/kaito/workspace/templates/qlora-params.yaml

```diff
@@ -29,7 +29,7 @@ data:
         bias: "none"
     
       TrainingArguments:
-        output_dir: "."
+        output_dir: "/mnt/results"
```

Comment:
Question about this is we should suppose it will have this path?

Reply:
Yes, so we are hardcoding this default path for now. This path is also default mounted by the sidecar container. In next PR we need to have basic parsing logic to make sure the output_dir specified here, is parsed, and the one the sidecar container also mounts.